.ci/eslint-plugin-zotero-translator/lib/rules/no-for-each.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+// this is a very simplistic rule to find 'for each' until I find a better eslint plugin that does this
+module.exports = {
+	meta: {
+		type: 'problem',
+		docs: {
+			description: 'disallow deprecated "for each"',
+			category: 'Possible Errors',
+		},
+	},
+
+	create: function (context) {
+		return {
+			Program: function (node) {
+				let lineno = 0;
+				let m;
+				for (const line of context.getSourceCode().getText().split('\n')) { // eslint-disable-line newline-per-chained-call
```

Comment:
This comment may no longer be necessary after I adjusted the ignore depth for that rule.

Reply:
Correct